server/src/utils/fight/fightMethods.ts

```diff
@@ -1355,6 +1388,18 @@ const breakShield = (fighter: DetailedFighter, opponent: DetailedFighter) => {
   return getFighterStat(fighter, 'disarm') * 100 >= randomBetween(1, 300);
 };
 
+const dropShield = (fightData: DetailedFight, fighter: DetailedFighter) => {
+  // Remove brute's shield
+  fighter.shield = false;
+  fighter.block -= SHIELD_BLOCK_ODDS;
```

Comment:
The way to handle shield drops have changed since your modifications, please update it. (skills array update + no more odds constant)